tests/integration/cases.yaml

```diff
@@ -0,0 +1,210 @@
+test_cases:
+  regular:
+  - id: ru
```

Comment:
I would prefer explicit names, also these are not IDs

Reply:
the purpose of these are a quick look up in the yaml file, should one of the tests fail + unique names for the pods. they are not suposed to describe anything and could be completely arbitrary, as long as they are unique.